experiment.py

```diff
@@ -83,8 +121,14 @@ def Scan_Field(stop):
         client.field.approach_mode.linear,
         client.field.driven_mode.driven
     )
+    time.sleep(100 * 60)  # temp ramp is limiting in time
+    #time.sleep(2500)
+    print(f"Field at {client.get_field()}")
+    print(f"Temp at {client.get_temperature()}")
+    
     
-    # TODO: Wait for dewar pressure below 4, delay 0
+    # Wait for dewar pressure
+    time.sleep(60 * 5)  # TODO: ask Bobby how long this takes!

```

Comment:
We should try to find how to wait for the dewar pressure using the client.

Reply:
Bobby answered this for us. It usually takes around 10 - 20 minutes. We will wait for an hour to be safe.